content/influxdb/v2.0/query-data/execute-queries/data-explorer.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Query in Data Explorer
+weight: 201
+menu:
+  influxdb_2_0:
+    name: Query with Data Explorer
+    parent: Execute queries
+influxdb/v2.0/tags: [query]
+---
+
+Queries can be built, executed, and visualized in InfluxDB UI's Data Explorer.
+
+![Data Explorer with Flux](/img/influxdb/2-0-data-explorer.png)
```

Comment:
I'd walk through the basics of building a query with the Data Explorer, then provide this link: 

https://docs.influxdata.com/influxdb/v2.0/visualize-data/explore-metrics/#explore-data-with-flux-and-the-data-explorer

I goes into more detail about using the Data Explorer.

Reply:
Hmm, this looks like the same procedure?
To simplify, prob makes sense to retitle "Explore metrics with InfluxDB" to "Query data in Data Explorer" and move over to Query data....

Reply:
I'd be fine with that. Just be sure to add the alias so any links point to the new location.